apps/backoffice-v2/src/pages/MerchantMonitoringUploadMultiple/hooks/useMerchantMonitoringUploadMultiplePageLogic/useMerchantMonitoringUploadMultiplePageLogic.tsx

```diff
@@ -68,6 +68,6 @@ export const useMerchantMonitoringUploadMultiplePageLogic = () => {
     onSubmit,
     onChange,
     csvTemplateUrl,
-    isSubmitting: isLoading,
+    showLoader: isLoadingCustomer || isSubmitting,
```

Comment:
I think we should change the naming to suggest the app state instead of the UI state. i.e. isCreateReportReady, name can be flexible if we negate the conditions.